agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -52,13 +62,54 @@ while true; do
 			shift;
 		fi
 		;;
+		--options)
+		display_sysinfo_opts
+		exit 0
+		;;
+		--sysinfo)
+		shift;
+		if [ -n "$1" ]; then
+			sysinfo="$1"
+			shift;
+		fi
+		;;
+		--check)
+		check=true
+		shift;
+		;;
 		--)
 		shift;
 		break;
 		;;
 	esac
 done
 
+if $check; then
+	for item in ${sysinfo//,/ }; do
+		if [ "$sysinfo" == "all" ] || [ "$sysinfo" == "default" ] || [ "$sysinfo" == "none" ]; then
```

Comment:
As it currently stands the following would be valid input:
```
pbench-user-benchmark --sysinfo=none,default,all,block
```
Do we want to allow that?  Perhaps we can move this check outside of the `for` loop, and only run the `for` loop if `sysinfo` is NOT one of those values?